### PR TITLE
GroupCommandTest classes

### PR DESCRIPTION
**Pull request by lelerer**

### CODE REVIEW

File: src/test/java/seedu/splitlah/command/GroupCreateCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import seedu.splitlah.data.Manager;
+import seedu.splitlah.exceptions.InvalidDataException;
+import seedu.splitlah.parser.Parser;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
```
Comment (ivanaitzliddat) [inline src/test/java/seedu/splitlah/command/GroupCreateCommandTest.java:11]: Consider removing extra new line

File: src/test/java/seedu/splitlah/command/GroupCreateCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+     */
+    @Test
+    public void run_validCommandWithPersonListDelimiter_groupIdIncremented() {
+        String userInput = "group /create /n grp2 /pl Cindy David";
+        Command command = Parser.getCommand(userInput);
+        int currentGroupId = manager.getProfile().getGroupIdTracker();
+        command.run(manager);
+        int testGroupId = manager.getProfile().getGroupIdTracker();
+        assertEquals(currentGroupId + 1, testGroupId);
+    }
```
Comment (ivanaitzliddat) [inline src/test/java/seedu/splitlah/command/GroupCreateCommandTest.java:55]: Possible to put these two together as groupSuccessfullyCreated

Comment (ivanaitzliddat) [reply src/test/java/seedu/splitlah/command/GroupCreateCommandTest.java:55]: Can also do so for the other tests (Can refer to activity create command test)

File: src/test/java/seedu/splitlah/command/GroupDeleteCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        // Check if group was successfully removed from the list of groups.
+        assertEquals(1, manager.getProfile().getGroupList().size());
+
+        // Check if the group still exists.
+        try {
+            manager.getProfile().getGroup(1);
+            fail();
+        } catch (InvalidDataException invalidDataException) {
+            assertEquals(Message.ERROR_PROFILE_GROUP_NOT_IN_LIST, invalidDataException.getMessage());
+        }
```
Comment (ivanaitzliddat) [inline src/test/java/seedu/splitlah/command/GroupDeleteCommandTest.java:51]: Good check, I will add this to activity delete as well

**Review by ivanaitzliddat — CHANGES_REQUESTED** [review]
Looks good overall, left some comments for group create command test

**Review by ivanaitzliddat — APPROVED** [review]
Looks good to merge

**Review by froststein — APPROVED** [review]
Looks good to merge

**Pull request merged**